dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java

```diff
@@ -117,24 +117,24 @@ class JdbcEventStore implements EventStore {
       " left join (select ri.eventid as ri_ev_id, json_agg(ri.relationshipid) as ev_rl FROM relationshipitem ri"
           + " GROUP by ri_ev_id)  as fgh on fgh.ri_ev_id=event.ev_id ";
 
-  private static final String EVENT_COMMENT_QUERY =
-      "select evc.eventid as evc_id,"
-          + " evnote.trackedentitycommentid as evnote_id,"
-          + " evnote.commenttext            as evnote_value,"
-          + " evnote.created                as evnote_storeddate,"
-          + " evnote.creator                as evnote_storedby,"
-          + " evnote.uid                    as evnote_uid,"
-          + " evnote.lastupdated            as evnote_lastupdated,"
-          + " userinfo.userinfoid            as evnote_user_id,"
-          + " userinfo.code                  as evnote_user_code,"
-          + " userinfo.uid                   as evnote_user_uid,"
-          + " userinfo.username              as evnote_user_username,"
-          + " userinfo.firstname             as evnote_user_firstname,"
-          + " userinfo.surname               as evnote_user_surname"
-          + " from eventcomments evc"
-          + " inner join trackedentitycomment evnote"
-          + " on evc.trackedentitycommentid = evnote.trackedentitycommentid"
-          + " left join userinfo on evnote.lastupdatedby = userinfo.userinfoid ";
+  private static final String EVENT_NOTE_QUERY =
+      "select evn.eventid as evn_id,"
+          + " n.noteid as note_id,"
+          + " n.notetext            as note_value,"
```

Comment:
```suggestion
          + " n.notetext            as note_text,"
```

Not sure why we used a different word like `value` instead of `text` and `stored` for `created` 🤷🏻 lets not do that as I don't see a reason why. We want less mappings in our 🧠